src/devices/LidarLiteV3/LidarLiteV3.cs

```diff
@@ -0,0 +1,319 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. 
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Iot.Device.TimeOfFlight.Models.LidarLiteV3;
+using System;
+using System.Buffers.Binary;
+using System.Device.Gpio;
+using System.Device.I2c;
+using System.Threading;
+
+namespace Iot.Device.TimeOfFlight
+{
+    /// <summary>
+    /// Lidar Lite v3 is a long-range fixed position distance sensor by Garmin.
+    /// </summary>
+    public class LidarLiteV3 : IDisposable
+    {
+        /// <summary>
+        /// Default address for LidarLiteV3
+        /// </summary>
+        public const byte DefaultI2cAddress = 0x62;
+
+        private GpioController _gpioController;
+        private I2cDevice _i2cDevice;
+        private int? _powerEnablePin;
```

Comment:
use `-1` for `no pin` and regular `int` (we've put that in our [guidelines](https://github.com/dotnet/iot/blob/master/Documentation/Devices-conventions.md#value-returned-by-sensor) - second bullet from the last)

Reply:
will fix -- also will try to address anything already mentioned in the guideline